components/src/core/info-list-item/info-list-item.tsx

```diff
@@ -374,8 +374,6 @@ export const InfoListItem: React.FC<InfoListItemProps> = (props) => {
     return (
         <TouchableOpacity
             accessible={true}
-            testID={`list-item-${title.replace(/\s+/g, '-').toLowerCase()}`}
-            accessibilityLabel={`list-item-${title.replace(/\s+/g, '-').toLowerCase()}`}
```

Comment:
I don't think we should be removing these at this time. I believe they were added for another team to be able to test, but this would also be a breaking change.